OneBlink.SDK/models/FormsAppBase.cs

```diff
@@ -0,0 +1,48 @@
+using System;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace OneBlink.SDK.Model
+{
+    public class FormsAppBase
+    {
+        private string[] AllowedTypes = new string[]
```

Comment:
Can this be an ENUM? then we dont need to validate the string in the setter.

Reply:
No, Enums have to be integral values https://stackoverflow.com/questions/8588384/how-to-define-an-enum-with-string-value

Reply:
Seems this is possible: https://stackoverflow.com/questions/2441290/javascriptserializer-json-serialization-of-enum-as-string

Reply:
worth doing IMO.

I wont disagree if you don't want to change it though :)